### PR TITLE
Civ 1978 timeline of events

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-1978

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by RosDiaf — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]
LGTM

File: src/main/modules/nunjucks/filters/dateFilter.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import {DateTime} from 'luxon';
```
Comment (laarmada) [inline src/main/modules/nunjucks/filters/dateFilter.ts:1]: Uncovered Lines on sonar

File: src/main/routes/urls.ts [hunk 2]
```diff
@@ -50,6 +50,7 @@ export const CITIZEN_REPAYMENT_PLAN = `${BASE_CASE_RESPONSE_URL}/full-admission/
 export const PARTIAL_ADMISSION_URL = `${BASE_CASE_RESPONSE_URL}/partial-admission`;
 export const CITIZEN_WHY_DO_YOU_DISAGREE_URL = `${PARTIAL_ADMISSION_URL}/why-do-you-disagree`;
 export const CITIZEN_TIMELINE_URL = `${BASE_CASE_RESPONSE_URL}/timeline`;
+export const CITIZEN_EVIDENCE_URL = `${BASE_CASE_RESPONSE_URL}/evidence`;
```
Comment (laarmada) [inline src/main/routes/urls.ts:53]: it's evidence no defendant/claimant-evidence?

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/routes/urls.ts (hunk 2 above)
Comment (TatsyMaslova) [reply src/main/routes/urls.ts:53]: it's evidence. and goes to rosario's screen. just like in ocmc

Comment (laarmada) [reply src/main/routes/urls.ts:53]: so sorry I couldn't find evidence screen in the old cmc.....

Comment (laarmada) [reply src/main/routes/urls.ts:53]: now!! sorry :)

**Review by laarmada — APPROVED** [review]

**Pull request merged**